moodle-cluster-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,665 @@
+    "publicDBIPAddressName": "[concat(parameters('namePrefix'),'DBIP')]",
+    "publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "publicDBIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicDBIPAddressName'))]",
+    "publicIPAddressType": "Dynamic",
+    "vmStorageAccountContainerName": "vhds",
+    "vmName": "[concat(parameters('namePrefix'),'VM')]",
+    "dataDiskUri": "[concat('http://',parameters('newStorageAccountName'),'.blob.core.windows.net/',variables('vmStorageAccountContainerName'),'/',parameters('namePrefix'),'-data.vhd')]",
+    "dataDiskSizeGB": "100",
+    "virtualNetworkName": "[concat(parameters('namePrefix'),'VNet')]",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+    "subnetRefDB": "[concat(variables('vnetID'),'/subnets/',variables('subnetNameDB'))]",
+    "availabilitySetName": "[concat(parameters('namePrefix'),'AvSet')]",
+    "lbName": "[concat(parameters('namePrefix'),'LB')]",
+    "lbDBName": "[concat(parameters('namePrefix'),'DBLB')]",
+    "lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('lbName'))]",
+    "lbDBID": "[resourceId('Microsoft.Network/loadBalancers',variables('lbDBName'))]",
+    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontend')]",
+    "frontEndDBIPConfigID": "[concat(variables('lbDBID'),'/frontendIPConfigurations/loadBalancerFrontend')]",
+    "lbProbeID": "[concat(variables('lbID'),'/probes/tcpProbe')]",
+    "lbDBProbeID": "[concat(variables('lbDBID'),'/probes/tcpProbe')]",
+    "lbPoolID": "[concat(variables('lbID'),'/backendAddressPools/LoadBalancerBackend')]",
+    "lbDBPoolID": "[concat(variables('lbDBID'),'/backendAddressPools/DBLoadBalancerBackend')]",
+    "SharedAzureFileName": "moodlefileshare",
+    "installFrontendScriptName": "install_frontend.sh",
+    "installFrontendScriptUri": "[concat('https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/moodle-cluster-ubuntu/', variables('installFrontendScriptName'))]",
+    "installFrontendCommand": "[concat('sh ', variables('installFrontendScriptName'), ' ', parameters('mySqlPassword'), ' ', parameters('newStorageAccountName'), ' ', variables('SharedAzureFileName'))]",
+    "installBackendScriptName": "install_backend.sh",
+    "installBackendScriptUri": "[concat('https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/moodle-cluster-ubuntu/', variables('installBackendScriptName'))]",
+    "installBackendCommand": "[concat('sh ', variables('installBackendScriptName'), ' ', parameters('mySqlPassword'), ' ', parameters('newStorageAccountName'), ' ', variables('SharedAzureFileName'))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```

Comment:
It's better to make apiVersion a variable for easier migration to newer API versions. Also use the GA API version 2015-06-15